### DISCUSSION TITLE
why would someones github account get suspended

### DISCUSSION THREAD

**Discussion by AyoubBenBrahim** [community/community#109211] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

i noticed people complaining about github account suspension which freaked me out, what reasons would lead to this

**Comment by MoldyLunchBox** [comment, marked as answer]
policy violations, fraudulent activities, abusive behavior, and security breaches...   GitHub accounts can be suspended for violating terms of service, community guidelines, or engaging in malicious activities such as spamming, intellectual property infringement, or fraud, while repeated violations or persistent disruptive behavior can lead to permanent suspension.
